src/System.Net.Primitives/ref/System.Net.Primitives.cs

```diff
@@ -472,6 +472,8 @@ public enum SslProtocols
         Tls = 192,
         Tls11 = 768,
         Tls12 = 3072,
+        [Obsolete("This value has been deprecated.  It is no longer supported. http://go.microsoft.com/fwlink/?linkid=14202")]
```

Comment:
Done. This suggests we should use more discretion when we add these to impls. I assume these are all okay: https://github.com/dotnet/corefx/pull/13771